uiowa/uiowa
# Facilities - maps for building pages

Resolves #6444 

# How to test
- `ddev blt frontend`
- `ddev blt ds --site=facilities.uiowa.edu `
- `ddev drush @facilities.local fm-buildings` (updates buildings to have lat/long coordinates, can take awhile to run)
- `ddev drush @facilities.local uli` 
- Check various buildings, see that map is rendering in iframe as expected. 
- Check iframe does not appear on buildings without lat/long values - example: https://facilities.uiowa.ddev.site/building/0013.
